build/deploy/grafana_dashboards/crdb-runtime-grafana.json

```diff
@@ -1819,11 +1819,11 @@
         "includeAll": false,
         "label": "Cluster name",
         "multi": false,
```

Comment:
and set this to value to "true" to enable it

Reply:
done